### ISSUE TITLE
Rule proposal: Missing parenthesis

### ISSUE THREAD

**Issue by zhangenming** [vuejs/eslint-plugin-vue#2129]
**Please describe what the rule should do:**

**What category should the rule belong to?**

[ ] Enforces code style (layout)
[ X ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
export default {
	props: {
		input: {
			type: Object,
			default: () => {} // unexpected return a void
		},
		output: {
			type: Object,
			default: () => ({})
		}
	}
}
```

**Additional context**

**Comment by ota-meshi** [comment]
Thank you for posting this issue.

However, I don't think it's a vue specific issue. I think you can use the no-empty-function rule.
https://eslint.org/docs/latest/rules/no-empty-function
